yt_dlp/extractor/facebook.py

```diff
@@ -420,6 +420,30 @@ def extract_metadata(webpage):
                 r'data-sjs>({.*?ScheduledServerJS.*?})</script>', webpage)]
             post = traverse_obj(post_data, (
                 ..., 'require', ..., ..., ..., '__bbox', 'require', ..., ..., ..., '__bbox', 'result', 'data'), expected_type=dict) or []
+            snippet = traverse_obj(post, (..., 'video', ..., 'attachments', ..., lambda k, v: (
+                k == 'media' and str(v['id']) == video_id and v['__typename'] == 'Video')), expected_type=dict) or {}
+            locale = self._html_search_meta(['og:locale', 'twitter:locale'], webpage, 'locale', default='en_US')
+            captions = get_first(snippet, ('video_available_captions_locales')) or get_first(snippet, ('captions_url')) or None
```

Comment:
I think this should be equivalent.
```suggestion
            captions = get_first(snippet, 'video_available_captions_locales', 'captions_url')
```